### DISCUSSION TITLE
Power Sensors of PoE Switches

### DISCUSSION THREAD

**Discussion by chris-kuhr** [Kane610/aiounifi#561] [Ideas]
Hi,

I would like to add the power metering of Unifi switches to HA. Would this be possible?
I would step in for that...
Best Ck

**Comment by Kane610** [comment]
AFAIK they only report poe consumption not the full consumption of the device itself. I guess investigating the data reported on the latest firmware of a poe switch and see if it's possible then I'm fine with adding it of course

**Reply by chris-kuhr** [reply]
do you have some heads up for the API documentation?
I only found this one so far:
https://github.com/Art-of-WiFi/UniFi-API-browser

**Reply by Kane610** [reply]
No you have to investigate yourself. To look at data output you can use this libraries mainfile
